app/model/editions/EditionsCard.scala

```diff
@@ -43,13 +42,14 @@ case class CardMetadata(
   replaceImage: Option[Image],
   overrideArticleMainMedia: Option[Boolean],
   coverCardImages: Option[CoverCardImages],
-  promotionMetric: Option[Double]
+  promotionMetric: Option[Double],
+  bio: Option[String] = None
```

Comment:
I think good to add under the direct list of `CardMetaData` as we have only "bio" for Chef card for now. Once we have more parameters to come under Chef and Recipe then may be we can have chefMetaData and RecipeMetaData?